evm/src/main/java/org/hyperledger/besu/evm/account/Account.java

```diff
@@ -66,7 +66,7 @@ default Optional<Address> delegatedCodeAddress() {
    *
    * @return true if the account has delegated code otherwise false.
    */
-  default boolean hasDelegatedCode() {
+  default boolean hasCodeDelegation() {
```

Comment:
very much a nit pick: I think I actually prefer hasDelegatedCode in this case, since it is clear the code is actually delegated. `hasCodeDelegation` could be read as "has code delegation feature but maybe it's not currently delegated"